### PR TITLE
Fixed Eth price issue on Transfer Eth screen

### PR DESCRIPTION
**Pull request by willbach**

### CODE REVIEW

File: packages/credit-protocol/index.ts [hunk 1]
```diff
@@ -436,14 +437,15 @@ export default class CreditProtocol {
 
   async getEthPrices() {
     const config = await this.getConfig()
+    console.log('ETH PRICES', config.ethereumPrices)
```
Comment (canterberry) [inline packages/credit-protocol/index.ts]: 🐛Left a `console.log()` here. 🔥

File: packages/reducers/app.ts [hunk 2]
```diff
@@ -105,15 +105,22 @@ export const pendingFriends = (state) => state.store.pendingFriends
...
 
 export const convertCurrency = (state) => (fromUcac: string, amount: number) : number => {
-  const fromExchange = Number(getEthExchange(state)(getUcacCurrency(state)(fromUcac)))
-  const toExchange = Number(getEthExchange(state)(defaultCurrency.toLowerCase()))
+  let fromExchange = Number(getEthExchange(state)(getUcacCurrency(state)(fromUcac)))
+  fromExchange = hasNoDecimals(getUcacCurrency(state)(fromUcac)) ? fromExchange : fromExchange * 100
+  let toExchange = Number(getEthExchange(state)(defaultCurrency.toLowerCase()))
+  toExchange = hasNoDecimals(defaultCurrency) ? toExchange : toExchange * 100
+
+  console.log(amount, fromExchange, toExchange)
```
Comment (canterberry) [inline packages/reducers/app.ts]: 🔥`console.log`

**Comment by arutun** [comment]
I think @canterberry is correct. I would expect the Release build to do away with any debug logging automatically, but it appears that is [not the case](https://medium.com/@jimgbest/i-have-been-building-react-native-rn-apps-for-quite-some-time-now-but-i-am-by-no-means-an-expert-8b258337561d) with console.log()...

**Comment by willbach** [comment]
Console.log statements removed

**Pull request merged**